lib/routes/charge/items/edit_item_page.dart

```diff
@@ -0,0 +1,316 @@
+import 'package:breez/bloc/account/account_actions.dart';
+import 'package:breez/bloc/account/account_bloc.dart';
+import 'package:breez/bloc/account/account_model.dart';
+import 'package:breez/bloc/account/fiat_conversion.dart';
+import 'package:breez/bloc/blocs_provider.dart';
+import 'package:breez/bloc/pos_catalog/actions.dart';
+import 'package:breez/bloc/pos_catalog/bloc.dart';
+import 'package:breez/bloc/pos_catalog/model.dart';
+import 'package:breez/bloc/user_profile/currency.dart';
+import 'package:breez/theme_data.dart' as theme;
+import 'package:breez/widgets/back_button.dart' as backBtn;
+import 'package:breez/widgets/flushbar.dart';
+import 'package:breez/widgets/single_button_bottom_bar.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+class EditItemPage extends StatefulWidget {
```

Comment:
What are the differences between AddItemPage and EditItemPage ?
It seems that by passing a model to the the AddItemPage you can differentiate the case and support edit with the same codebase.
Thoughts?

Reply:
I agree, we should reduce it to one page. If an item + model is passed to the page, it should be in edit mode(initialize the fields and use UpdateItem action).

Reply:
[Done](https://github.com/breez/breezmobile/pull/296/commits/593e9e2cdf47bad76806d4d97fd740ca12b1caff )
[bugfix](https://github.com/breez/breezmobile/pull/296/commits/dd8bf2e210865627140b00cbfbe93ec07dfa1219)

Reply:
@roeierez Replaced _selectedCurrency & _selectedFiatCurrency with [CurrencyWrapper](https://github.com/breez/breezmobile/pull/296/commits/01547fd5640ce7fb6c51732c6560b38bb11e4277)